libraries/AP_ExternalAHRS/AP_ExternalAHRS_InertialLabs.cpp

```diff
@@ -398,21 +406,53 @@ bool AP_ExternalAHRS_InertialLabs::check_uart()
         }
         case MessageType::SUPPLY_VOLTAGE: {
             CHECK_SIZE(u.supply_voltage);
-            state2.supply_voltage = u.supply_voltage*0.01;
+            state2.supply_voltage = u.supply_voltage*0.01; // V
             break;
         }
         case MessageType::TEMPERATURE: {
             CHECK_SIZE(u.temperature);
             // assume same temperature for baro and airspeed
-            baro_data.temperature = u.temperature*0.1;
-            airspeed_data.temperature = u.temperature*0.1;
+            baro_data.temperature = u.temperature*0.1; // degC
+            airspeed_data.temperature = u.temperature*0.1; // degC
+            ins_data.temperature = u.temperature*0.1;
             break;
         }
         case MessageType::UNIT_STATUS2: {
             CHECK_SIZE(u.unit_status2);
             state2.unit_status2 = u.unit_status2;
             break;
         }
+        case MessageType::GNSS_ANGLES: {
+            CHECK_SIZE(u.gnss_angles);
+            gnss_data.heading = u.gnss_angles.heading*0.01; // deg
+            gnss_data.pitch = u.gnss_angles.pitch*0.01; // deg
```

Comment:
`pitch` in `gnss_data`?!

I think there might be a reason it was generically called `state2`?

Reply:
I try to use the same terms as in the device documentation
For consistency and simplifying code maintenance
![image](https://github.com/user-attachments/assets/7755da0c-7dfe-48fd-833a-f49985e8eca9)

*I can rename, if need it